README.md

```diff
@@ -1,2 +1,175 @@
+               eyes.check("Home Page", Target.window().strict()
+               .layout(By.xpath("//div[@class='ReactVirtualized__Grid items']"),
+                  By.id("breadcrumb-container"),
+                  By.className("//div[@class='  breadcrumb-section']"))
+               .ignore(By.xpath("//div[@class='component-layout']//div[@class" + "='img-animate'][1]"),
+                  By.xpath("//div[@class='banner-components-runtime']/div")));
+               ```
+            2. For more detailed explanation and different examples, visit
+               URL: [The Eyes SDK check Fluent API](https://applitools.com/docs/topics/sdk/the-eyes-sdk-check-fluent-api.html)
+    6. #### Using Ultra Fast Grid:
+       Applitools Ultrafast Grid allows you to perform visual testing across multiple browsers and devices in seconds,
+       to make sure that your content is visually perfect on every device, screen size, and browser combination. To Run
+       Visual Validations on multiple Browser Types Using Ultra Fast Grid, Set `USE_UFG` to `true`
+       in `applitools.properties` file of your automation framework repo. Browser Combination need to be added
+       in `setUFGBrowserConfig(Configuration eyesConfig)`
+       in `src/main/java/com/znsio/api/ApplitoolsConfigurationManager.java` as illustrated in Below example:
+        ```
+        eyesConfig.addBrowser(1512, 866, BrowserType.CHROME);
+        eyesConfig.addBrowser(1600, 1200, BrowserType.FIREFOX);
+        eyesConfig.addBrowser(1024, 768, BrowserType.SAFARI);
+        eyesConfig.addDeviceEmulation(DeviceName.Galaxy_Note_10);
+        eyesConfig.addDeviceEmulation(DeviceName.iPhone_11_Pro_Max);
+        ```
+       For More Details on Configuring Ultra Fast Grid,
+       Visit: [Ultrafast Grid configuration](https://applitools.com/docs/topics/sdk/vg-configuration.html)
+    7. #### Setting correct Viewport Size:
+        1. Incorrect Viewport size will cause the tests to be skipped with exception
+           as `EyesException: Unable to set Viewport Size`.
+        2. In that case, follow the below process to set Viewport Size Correctly:
+            - Navigate to Site: `https://whatismyviewport.com/` to get the viewport size.
+            - Set the Viewport size in `VIEWPORT_SIZE` in format: `Widthxheight` in `applitools.properties` file of your
+              automation framework repo
+
+3. All the Applitools Batch, Config and UFG Browser configuration related code is present in file `
+   src/main/java/com/znsio/api/ApplitoolsConfigurationManager.java``.
+    1. To add Batch Properties, navigate to above java class and add code in
+       method: `setBatchProperties(BatchInfo batchInfo)` as per given syntax and example:
+       ```
+       batchInfo.addProperty(<Key>, <Value>);
+       ```
+       example:
+       ```
+       batchInfo.addProperty("Agent Name", System.getenv("AGENT_NAME"));
+       ```
+    2. To add Test Properties, navigate to above Java class and add code in
```

Comment:
The consumer of this utility will not have access to the ApplitoolsConfigurationManager to make changes to it

Reply:
This was based on earlier implementation. Updated the wording to not confuse the consumer. Still kept it there to help consumers understand how are we configuring these properties.

Reply:
As discussed, removed this.